### ISSUE TITLE
atom-text-editor.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by edarblanco** [thomaslindstrom/color-picker#209]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
atom-text-editor.Object.defineProperty.get (/usr/lib/atom/src/text-editor-element.js:68:18)
Object.open (/home/edar/.atom/packages/color-picker/lib/ColorPicker-view.coffee:319:32)
atom-text-editor.<anonymous> (/home/edar/.atom/packages/color-picker/lib/ColorPicker.coffee:36:29)
CommandRegistry.handleCommandEvent (/usr/lib/atom/src/command-registry.js:259:29)
handleCommandEvent (/usr/lib/atom/src/command-registry.js:3:59)
CommandRegistry.dispatch (/usr/lib/atom/src/command-registry.js:160:19)
```

**Comment by RenaldasK** [comment]
Same issue here.

**Comment by vineeth1991** [comment]
Same here

**Comment by aslami** [comment]
Same here

**Comment by Oblik-Design** [comment]
same here too.

**Comment by SpectralKH** [comment]
Yep, same

**Comment by aradabir007** [comment]
same

**Comment by dirkdirk** [comment]
M2

**Comment by Provice911** [comment]
same

**Comment by Sokuto** [comment]
same what do

**Comment by thomaslindstrom** [comment]
`color-picker` still works, right, or is it not opening? I'll probably fix this deprecation warning by the end of the week.

**Comment by Oblik-Design** [comment]
Great, yep still works, most package are up to speed now, pressure pressure !! ;)

**Comment by dirkdirk** [comment]
Still working for me.  You've got time to fix it.  It's 'just' a deprecation warning, not a fatal error.

Thanks for a great tool!

**Comment by thomaslindstrom** [comment]
Hi, guys. This should be fixed in `v2.2.5`.
